docs/release-notes/2.37.0.md

```diff
@@ -0,0 +1,164 @@
+# Percona Monitoring and Management 2.37.0
+
+| **Release date:** | April 24, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+ --->
+
+
+## Release Highlights
+
+
+### Improved Inventory view
+
+This release of PMM starts the series of enhancements that will improve the way you work with Services and Nodes in PMM. 
+With this first set of changes, we’re revamping the **Inventory** page to give you more context on your inventory objects, and more actionable information for exploring and fixing possible issues. 
+
+Here are all the specific changes: 
+
+- Improved design of the **Inventory** page for a clearer presentation of all the objects registered by PMM. 
+- Services now show their database status, which is based on the monitoring metrics. This enables you to quickly check the status of the databases when you are working with the inventory.  
+- New **Monitoring** column showing the live status for the monitoring Agents assigned to a Service. This enables you to quickly spot any possible issues with the Agents.
+- Quick access to the Service Summary **Dashboard** and to **QueryAnalytics** results so that you can easily explore important related information. 
+- New **Details** section grouping together all the relevant parameters and labels for a Service. To keep things clean, this section is collapsed by default, so that you can view extra information only when you need it. 
+
+Here's what's coming:
+
+Building up on these changes, subsequent releases will further improve PMM Inventory with:
+
+- More useful features to filter and better connect Services and Nodes.
+- Services organized in clusters so that you can capture the bigger picture, instead of working at of individual Service-level.
+
+### New Experimental Dashboards for Kubernetes Monitoring
+
+!!! caution alert alert-warning "Important"
+    These experimental dashboards are subject to change. It is recommended to use these dashboards for testing purposes only.
+
+We are pleased to announce the release of PMM V2.37, which introduces three new experimental dashboards:
```

Comment:
Lowercase v2.37? Not sure though :)

Reply:
Updated.

Reply:
Did you? :)

Reply:
https://github.com/percona/pmm-doc/pull/1043/files#diff-b1a89c26b922087ff41c5103950e64f8b8e3a8c4187c770a43ca003596444449R46